src/main/webapp/search_area.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+async function getListOfOrganizations() {
+  const response = await fetch(`/list-organizations`);
+  const organizations = await response.json();
+
+  return organizations;
+}
+
+document.addEventListener("DOMContentLoaded", async function() {
+  const organizations = await getListOfOrganizations();
+
+  // TODO: Get Maintainer status by checking if requester User is a Maintainer. 
+  // This checks that requester User has valid credentials to edit/delete/view ALL organizations. (by checking userID)
+  let isMaintainer = false;
+  if (document.getElementById('search-area')) {
```

Comment:
These bits (if search-area is in the page vs if all-organizations is in the page) should be in page-specific bits of javascript.

Reply:
I put a TODO comment to fix this when we create separate JS files for "Help Near Me" page vs "Organizations" page. Right now we have one JS class that is used in both pages.